medicines/doc-index-updater/examples/manifests/README.md

```diff
@@ -50,52 +46,44 @@ kubectl create secret generic redis-key \
 
 #### Create Queue Key
 
-- retrieve the keys for the service bus access policy from the Azure Portal or the CLI:
+- retrieve the keys for the service bus access policy from the Azure Portal or the CLI, and create the sealed secret with one of the above keys:
 
 ```bash
-az servicebus queue authorization-rule keys list \
+kubectl create secret generic create-queue-policy-key \
+ -n doc-index-updater \
+ -o json \
+ --dry-run \
+ --from-literal key=$(
+   az servicebus queue authorization-rule keys list \
   --resource-group MHRA-dev \
   --namespace-name doc-index-updater-dev \
   --queue-name doc-index-updater-create-queue \
   --name doc-index-updater-create-auth \
   --query primaryKey \
   --output tsv
-```
-
-- create the sealed secret with one of the above keys (replace `<insert key here>` with the key):
-
-```bash
-kubectl create secret generic create-queue-policy-key \
- -n doc-index-updater \
- -o json \
- --dry-run \
- --from-literal key=<insert key here> \
+ ) \
 | kubeseal \
  --format yaml > SealedSecret-create-queue-key.yaml
 ```
 
 #### Delete Queue Key
 
-- retrieve the keys for the service bus access policy from the Azure Portal or the CLI:
-
-```bash
-az servicebus queue authorization-rule keys list \
-  --resource-group MHRA-dev \
-  --namespace-name doc-index-updater-dev \
-  --queue-name doc-index-updater-delete-queue -\
-  -name doc-index-updater-delete-auth \
-  --query primaryKey \
-  --output tsv
-```
-
-- create the sealed secret with one of the above keys (replace `<insert key here>` with the key):
+- retrieve the keys for the service bus access policy from the Azure Portal or the CLI, and create the sealed secret with one of the above keys:
```

Comment:
again, user of these docs just wants something that creates the keys, isn't interested in how they're retrieved necessarily

Reply:
as above